src/lib/rest/rest.cpp

```diff
@@ -240,7 +240,7 @@ static int uriArgumentGet(void* cbDataP, MHD_ValueKind kind, const char* ckey, c
   {
     containsForbiddenChars = forbiddenChars(val, ";");
   }
-  else if ((key != "q") && (key != "idPattern"))
+  else if ((key != "q") && (key != "mq") && (key != "idPattern"))
```

Comment:
Fixed in f3b2e4a

Only q and mq - to not mix things too much in this PR.
There are plenty of other URI params (like idPattern) that use simple strings. Most of them don't even have a define in uriParamNames.h. This 'problem' should probably be addressed in another PR.